docs/source/design/github_actions.rst

```diff
@@ -0,0 +1,84 @@
+.. -*- mode: rst -*-
+.. vi: set ft=rst sts=4 ts=4 sw=4 et tw=79:
+
+.. _chap_design_github_action:
+
+*******************************
+GitHub Action
+*******************************
+
+.. topic:: Specification scope and status
+
+   This specification describes a proposed interface to a DataLad GitHub Action.
+
+The purpose of the DataLad GitHub Action is to support CI testing with DataLad datasets
+by making it easy to install ``datalad`` and ``get`` data from the dataset.
+
+
+Example Usage
+=============
+
+Dataset installed at ``${GITHUB_WORKSPACE}/studyforrest-data-phase2``,
+``get``'s all the data::
```

Comment:
are you sure that getting all the data should be default?  E.g. in this demo use case it would be 14GB! (will CI have enough time to fetch it all? ;-))

Reply:
My use case involves a dedicated small unit testing dataset, but, yes I can see how result in a large download :-).